client/src/views/organizations/creation/OrganizationSummaryPage.vue

```diff
@@ -41,7 +51,7 @@
             {{ $msTranslate('CreateOrganization.overview.sequester') }}
```

Comment:
This title is still `Sequester key` / `Clé de séquestre`. I'd change it either to `Data sequester` / `Séquestre des données` to match the toggle field in the first step , or to `Authority key` / `Clé d'autorité`.